src/entities/encoders/composableStable.ts

```diff
@@ -22,6 +25,78 @@ export class ComposableStableEncoder {
         // eslint-disable-next-line @typescript-eslint/no-empty-function
     }
 
+    /**
+     * Encodes the User Data for adding liquidity to a ComposableStablePool
+     * @param kind Kind of the Add Liquidity operation: Init, Unbalanced, SingleToken, Proportional
+     * @param amounts Amounts of tokens to be added to the pool
+     * @returns
+     */
+    static encodeAddLiquidityUserData(
+        kind: AddLiquidityKind,
+        amounts: AddLiquidityAmounts & { maxAmountsInNoBpt: bigint[] },
```

Comment:
We've been using `withBpt` and `withoutBpt` - can you please update to keep the standard.